### PR TITLE
Azure-theme pivots + dropdowns for 7.0 

### PR DESCRIPTION
**Pull request by Jacqueline-ms**
#### Pull request checklist

- [ x ] Addresses an existing issue: Fixes DevOps # 7692163
- [ x ] Include a change request file using `$ yarn change`

### Dropdowns 
- Removed control outline via callout, updated separator line for each theme
- Per Figma designs: dropdown input 24px and dropdown items 32px 

### Callout
- implementation vs figma 
<div style="display:flex">

</div>

### Label 
- removed semibold from label per label redesign 

### Pivots
- Removed left/right padding from individual pivot
- Fixed indentation of Pivot on rest state, aligned with rest of content on the page.

### CODE REVIEW

File: packages/azure-themes/src/azure/styles/Callout.styles.ts [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 export const CalloutContentStyles = (props: ICalloutContentStyleProps): Partial<ICalloutContentStyles> => {
   const { theme } = props;
   const { semanticColors } = theme;
 
   return {
     root: {
-      borderColor: semanticColors.inputBorder,
-      borderStyle: StyleConstants.borderSolid,
-      borderWidth: StyleConstants.borderWidth,
-      boxShadow: Depths.depth8,
```
Comment (wsmd) [inline packages/azure-themes/src/azure/styles/Callout.styles.ts:14]: This also affect the `Callout` component, but I didn't see it mentioned in the PR description? Is this change intended? 
 
| Before | After |
| --- | --- |
| Screen Shot 2020-10-07 at 1 39 15 PM | Screen Shot 2020-10-07 at 1 38 05 PM |

**Review by wsmd — COMMENTED** [review]
Nice work @Jacqueline-ms! I left a comment about a change that I didn't see included in the PR description so just wanted to make sure it's intended. Otherwise, this is good to go!

File: packages/azure-themes/src/azure/styles/Callout.styles.ts (hunk 1 above)
Comment (Jacqueline-ms) [reply packages/azure-themes/src/azure/styles/Callout.styles.ts:14]: @wsmd great point, I left out that screenshot. All of our dialogs / dropdowns have no border now. I will reduce the dropshadow and include image:


## Figma design

**Review by wsmd — APPROVED** [review]
@Jacqueline-ms Thanks for doing this!
